2.13/tests/test/test.js

```diff
@@ -432,6 +432,42 @@ describe('DELETE /v2/service_instance/:instance_id', function() {
     });
 });
 
+function testInstanceLastOperationStatus(instance_id, done) {
+    endpoint = '/v2/service_instances/' + instance_id + '/last_operation'
+
+    testAPIVersionHeader(endpoint, 'GET');
+    testAuthentication(endpoint, 'GET');
+
+    var pollDuration = 0;
+    var succeededOrFailed = false;
+    var interval = setInterval(function() {

```

Comment:
Replacing `setInterval` with [`async/whilst`](https://caolan.github.io/async/docs.html#whilst) is more graceful IMO. I propose to widely use the async package in OSB checker. It can be helpful to interact with service brokers by nodejs.